### PR TITLE
change makeStyles to styled in AppBarBase and DrawerBase

### PR DESCRIPTION
**Pull request by kien123456k**
This PR is for change AppBarBase and DrawerBase from using makeStyles API to styled API in Material UI v5.
Please review it for me. Thank you!

### CODE REVIEW

File: src/components/AppBars/AppBarBase/AppBarBase.stories.tsx [hunk 1]
```diff
@@ -17,17 +17,21 @@ export default {
...
+export const Basic: Story<AppBarBaseProps> = (args) => {
     return (
         <AppBarBase {...args}>
             <ToolbarBase>
                 <DrawerTriggerButton>
                     <MenuIcon />
                 </DrawerTriggerButton>
-                <TypographyBase variant="h6" noWrap>
-                    Simple App Bar Drawer
+                <TypographyBase variant="h6" component="div" sx={{ flexGrow: 1 }}>
```
Comment (binh1298) [inline src/components/AppBars/AppBarBase/AppBarBase.stories.tsx:27]: why do you want it to be a div? Why usse flexGrow?

File: src/components/Drawers/DrawerBase/index.tsx [hunk 2]
```diff
@@ -12,58 +8,57 @@ export interface DrawerBaseProps extends MaterialDrawerProps {
...
+        easing: theme.transitions.easing.sharp,
+        duration: theme.transitions.duration.leavingScreen,
+    }),
+    width: `calc(${theme.spacing(8)} + 1px)`,
+    [theme.breakpoints.up("sm")]: {
+        width: `calc(${theme.spacing(9)} + 1px)`,
+    },
+});
+
+const Drawer = styled(MaterialDrawer, {
```
Comment (binh1298) [inline src/components/Drawers/DrawerBase/index.tsx:30]: I don't think I understand this, can we join a discord and talk about it?

File: src/components/AppBars/AppBarBase/AppBarBase.stories.tsx (hunk 1 above)
Comment (kien123456k) [reply src/components/AppBars/AppBarBase/AppBarBase.stories.tsx:27]: Actually, I don't know. I see it in the Basic App Bar in Mui document. You can see the image below.
Here is the link: https://mui.com/components/app-bar/
image

File: src/components/Drawers/DrawerBase/index.tsx (hunk 2 above)
Comment (kien123456k) [reply src/components/Drawers/DrawerBase/index.tsx:30]: Okay, I will contact with you through messager.

**Review by binh1298 — APPROVED** [review]

**Pull request merged**